crates/driver/src/infra/api/mod.rs

```diff
@@ -45,14 +56,24 @@ impl Api {
                 solver,
                 shared: Arc::clone(&shared),
             });
-            app = app.nest(&name.0, router);
+            app = app.nest(&format!("/{}", name.0), router);
```

Comment:
Should we make `name: Display` and change this to:
```suggestion
            app = app.nest(&format!("/{}", name), router);
```

Reply:
Why not `format!("{name}")`.